### ISSUE TITLE
Add time picker 

### ISSUE THREAD

**Issue by aaronucsd** [clauderic/react-infinite-calendar#7]
Can we add time picker with the date?

**Comment by bradennapier** [comment]
+1

**Comment by larafale** [comment]
+1

**Comment by kevinrademan** [comment]
+1

**Comment by mogarick** [comment]
Any news about this feature request? Is it viable or it's something totally out of consideration for the rest of the year at least?

**Comment by clauderic** [comment]
It's not completely out of the cards, I'm just not sure what the UI should look like for the time picker. Part of me also thinks this is a little out of scope, but I'd be open to considering it.

Most of the material design-esque time pickers I've seen look like this, and are a bit clunky to use from a user-experience perspective.

![components_pickers_time1](https://cloud.githubusercontent.com/assets/1416436/25881021/b18b3d14-3508-11e7-8f02-baf14a5a8efd.png)

**Comment by bvanbrem** [comment]
Hi! As i'm interested in your project, I would like to do a suggestion for the timepicker UI.  Please look at a more detailed description in the attachements.  Feel free to use me if you think I can help you with user behaviour and other project related (not code related) issues.

![timepickersimple](https://cloud.githubusercontent.com/assets/8875724/25899089/d150633c-358e-11e7-96e2-3479858f277b.PNG)
[time picker.pptx](https://github.com/clauderic/react-infinite-calendar/files/989741/time.picker.pptx)

**Comment by knordstrom** [comment]
I agree on the Material clock design, the UX is painful. Particularly because the mechanical clock is not a common meme these days and younger generations are less familiar with it.

I like the simplicity of what @bvanbrem proposed. Though configurable to include seconds.

To launch it, I was imagining something whereby a bubble/icon is available above each selected date that could be clicked to open a time picker. The time would be restricted to the 24 hour period for that day (to avoid handling for overflow).

This is a great project, perhaps the nicest I've seen from a UX perspective - innovative while remaining intuitive. Would love to use it in work we're doing but I'm afraid time selection is critical to our use case. Any thoughts on when this might be in progress, if at all?

**Comment by email2vimalraj** [comment]
Anyone working on this feature? It will be needed by many people (being selfish :) )

I'll see if I can also contribute.

**Comment by dilizarov** [comment]
Any work on this front, @clauderic. I'd be willing to contribute.

**Comment by danbovey** [comment]
For anybody like me looking for a timeslots feature (where you view the week horizontally and time throughout the days vertically), see [react-big-calendar](https://github.com/intljusticemission/react-big-calendar). 

http://intljusticemission.github.io/react-big-calendar/examples/index.html

**Comment by kesha-antonov** [comment]
Since it does not have Time picker I'm switching to another package :(

**Comment by sflahave** [comment]
I like Ant Design's TimePicker component as a design guide - it's more compact than Material-UI time picker with it's skeumorphic analog clock design. https://ant.design/components/time-picker/
